src/type/preload.ts

```diff
@@ -508,92 +508,101 @@ export type ConfirmedTips = {
-      fixedExportDir: z.string().default(""),
-      exportLab: z.boolean().default(false),
-      exportText: z.boolean().default(false),
-      outputStereo: z.boolean().default(false),
-      audioOutputDevice: z.string().default(""),
-    })
-    .default({}),
-  hotkeySettings: hotkeySettingSchema.array().default(defaultHotkeySettings),
-  toolbarSetting: toolbarSettingSchema
-    .array()
-    .default(defaultToolbarButtonSetting),
-  engineSettings: z.record(engineIdSchema, engineSettingSchema).default({}),
-  userCharacterOrder: speakerIdSchema.array().default([]),
-  defaultStyleIds: z
-    .object({
-      engineId: engineIdSchema
-        .or(z.literal(EngineId("00000000-0000-0000-0000-000000000000")))
-        .default(EngineId("00000000-0000-0000-0000-000000000000")),
-      speakerUuid: speakerIdSchema,
-      defaultStyleId: styleIdSchema,
-    })
-    .array()
-    .default([]),
-  presets: z
-    .object({
-      items: z
-        .record(
-          presetKeySchema,
-          z.object({
-            name: z.string(),
-            speedScale: z.number(),
-            pitchScale: z.number(),
-            intonationScale: z.number(),
-            volumeScale: z.number(),
-            prePhonemeLength: z.number(),
-            postPhonemeLength: z.number(),
-            morphingInfo: z
-              .object({
-                rate: z.number(),
-                targetEngineId: engineIdSchema,
-                targetSpeakerId: speakerIdSchema,
-                targetStyleId: styleIdSchema,
-              })
-              .optional(),
-          })
-        )
-        .default({}),
-      keys: presetKeySchema.array().default([]),
-    })
-    .default({}),
-  defaultPresetKeys: z.record(voiceIdSchema, presetKeySchema).default({}),
-  currentTheme: z.string().default("Default"),
+// ルート直下にある雑多な設定値
+export const RootMiscSettingSchema = z.object({
   editorFont: z.enum(["default", "os"]).default("default"),
   showTextLineNumber: z.boolean().default(false),
   showAddAudioItemButton: z.boolean().default(true),
-  experimentalSetting: experimentalSettingSchema.default({}),
-  acceptRetrieveTelemetry: z
-    .enum(["Unconfirmed", "Accepted", "Refused"])
-    .default("Unconfirmed"),
-  acceptTerms: z
-    .enum(["Unconfirmed", "Accepted", "Rejected"])
-    .default("Unconfirmed"),
   splitTextWhenPaste: z
     .enum(["PERIOD_AND_NEW_LINE", "NEW_LINE", "OFF"])
     .default("PERIOD_AND_NEW_LINE"),
   splitterPosition: splitterPositionSchema.default({}),
-  confirmedTips: z
-    .object({
-      tweakableSliderByScroll: z.boolean().default(false),
-      engineStartedOnAltPort: z.boolean().default(false),
-      notifyOnGenerate: z.boolean().default(false),
-    })
-    .default({}),
-  registeredEngineDirs: z.string().array().default([]),
-  recentlyUsedProjects: z.string().array().default([]),
   enableMultiEngine: z.boolean().default(false),
 });
+export type RootMiscSetting = z.infer<typeof RootMiscSettingSchema>;
```

Comment:
```suggestion
export type RootMiscSettingType = z.infer<typeof RootMiscSettingSchema>;
```
configと揃えてこうしたほうがいいような気がします。

Reply:
確かに感あるのですが、`ExperimentalSetting`などは`Type`と書かれてないのと、`Config`はクラス名とどっちがどっちか分からないから`Type`と書いた方がいいのとがあって、ここはこのままの方が良いかもと思いました！

Reply:
なるほど。あとで統一したいですね...

Reply:
リファクタリングがしがしやって行くの結構楽しいと思うので、もしよかったら暇つぶしにぜひ！！
（コーディング力がすごい速度で上がると思います）